### DISCUSSION TITLE
Sketchware affected with PUA

### DISCUSSION THREAD

**Discussion by PrakharDoneria** [Sketchware-Pro/Sketchware-Pro#434] [General]
If you download Sketchware [Latest Version(both realises)](https://github.com/Sketchware-Pro/Sketchware-Pro/releases/tag/v6.4.0-rc03) they have some PUA problems that can effect your devices. 
They report is available with this issue and was released by Jio Security Powered by [Norton Life Lock](https://github.com/nortonlifelock) 

Refer the screenshot below for more-
![Screenshot_20220603-141136](https://user-images.githubusercontent.com/92568876/171821844-3a79aaa3-aaf4-4995-9022-1a61bbf66c27.png)
![Screenshot_20220603-141132](https://user-images.githubusercontent.com/92568876/171821853-ba653dcf-8a12-43ae-a137-6d5cdacc1640.png)
![Screenshot_20220603-141123](https://user-images.githubusercontent.com/92568876/171821858-482a0252-48a2-4754-b776-5bbf364654f4.png)

**Comment by j-px** [comment]
PUA stands for **Potentially** unwanted app. That doesn't have to mean it 100 % is an unwanted app. It probably got flagged as it's building APKs on demand.

**Comment by ReduxFlakes** [comment]
Norton is not even a good product and company since they known to do some sketchy stuff with their user data and their not great scanner and PUA is not really an issue. Try Malwarebytes and see if it detects Sketchware as a PUA. More info here: https://en.wikipedia.org/wiki/Potentially_unwanted_program
